src/dicomweb_client/api.py

```diff
@@ -141,6 +141,8 @@ def set_http_retry_params(
         retry: bool = True,
         max_attempts: int = 5,
         wait_exponential_multiplier: int = 1000,
+        wait_random_min: int = None,
```

Comment:
Can we remove the `random` from the parameter names and just use `wait_min` and `wait_max` rather than `wait_random_min` and `wait_random_max`, respectively?